dev-python/matplotlib2tikz/matplotlib2tikz-0.6.6.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
```

Comment:
Nevermind that. Turns out deps don't support 3.6 anyway, and some of them have broken tests.